pom.xml

```diff
@@ -150,89 +80,6 @@
     </dependency>
   </dependencies>
 
-  <build>
```

Comment:
Usikker på konsekvensen av å fjerne hele build-gruppa...

Reply:
Jeg også, det ser ut til å fungere, men vet ikke om det påvirker hvilken måte klassene bygges på.

Reply:
Foreslår å kopiere koden vi trenger fra bidrag-commons (CorrelationId) for å slippe avhengigheten hvis ikke @ugur93 har tenkt å splitte den opp